tests/test_initialize_epoch.py

```diff
@@ -1,14 +1,80 @@
 # ensure that our fixture 'new_epoch' functions properly
-def test_new_epoch(casper_chain, casper, new_epoch):
+def test_new_epoch_fixture(casper_chain, casper, new_epoch):
+    epoch_length = casper.EPOCH_LENGTH()
     for _ in range(4):
         prev_epoch = casper.current_epoch()
         prev_block_number = casper_chain.head_state.block_number
-        expected_jump = casper.EPOCH_LENGTH() - (prev_block_number % casper.EPOCH_LENGTH())
+        expected_jump = epoch_length - (prev_block_number % epoch_length)
 
         new_epoch()
 
+        block_number = casper_chain.head_state.block_number
         assert casper.current_epoch() == prev_epoch + 1
-        assert casper_chain.head_state.block_number == prev_block_number + expected_jump
+        assert block_number == prev_block_number + expected_jump
+        assert (block_number % epoch_length) == 0
+
+
+def test_epoch_length_range(casper_chain, casper, new_epoch, assert_tx_failed):
+    epoch_length = casper.EPOCH_LENGTH()
+
+    new_epoch()
+
+    for _ in range(0, epoch_length):
```

Comment:
Range start defaults to 0 unless you specify otherwise. `range(epoch_length)` is fine here.